cpp/find_latest_journal_year.cc

```diff
@@ -183,6 +190,19 @@ void ProcessRecords(MARC::Reader * const marc_reader, std::unordered_map<std::st
     LOG_INFO("Processed " + std::to_string(record_count) + " MARC record(s).");
 }
 
+std::string explode(const std::string &data, const std::string &delimiters) {
```

Comment:
Same here, please just use StringUtil::Split() instead of writing a new explode function.
(Another note: If you ever have any need for such a general function, please implement it within a re-usable library instead of adding it to a single script.)